concordium-std-derive/README.md

```diff
@@ -1,3 +1,6 @@
+
+**Deprecated: The content of this crate have been moved to the crate concordium-contracts-common-derive**
```

Comment:
This is OK for now, but once we publish the new versions of all the libraries we should remove this package from the repository.